apps/i18n/common/en_us.json

```diff
@@ -328,35 +335,25 @@
   "showToolbox": "Show Toolbox",
   "showVersionsHeader": "Version History",
   "signup": "Sign up for the intro course",
+  "stageLocked": "Stage is locked.",
+  "stageNotFullyLocked": "Stage is not locked for all.",
   "stageRewards": "Stage Rewards",
+  "startLearning": "Start learning.",
   "stepIn": "Step in",
-  "stepOver": "Step over",
   "stepOut": "Step out",
+  "stepOver": "Step over",
   "stringEquals": "string=?",
   "student": "Student",
+  "studentControl": "Individual student control",
   "studentLockStateInstructions": "Use the table below to control the state for each individual student.",
   "studentName": "Student Name",
   "submit": "Submit",
   "submitYourProject": "Submit your project",
   "submitYourProjectConfirm": "You cannot edit your project after submitting it, really submit?",
-  "teacherPanel": "Teacher Panel",
-  "tryNow": "Try Now",
-  "unplugged": "UNPLUGGED",
-  "unpluggedActivity": "UNPLUGGED ACTIVITY",
-  "usingHints": "Using hints",
-  "usingTooManyBlocks": "Using too many blocks",
-  "unsubmit": "Unsubmit",
-  "unsubmitYourProject": "Unsubmit your project",
-  "unsubmitYourProjectConfirm": "Unsubmitting your project will reset the submitted date, really unsubmit?",
-  "showAnswers": "Show answers",
-  "showAnswersInstructions": "\"Show answers\" to put the assessment into a read-only mode.",
-  "stageLocked": "Stage is locked.",
-  "stageNotFullyLocked": "Stage is not locked for all.",
-  "student": "Student",
-  "studentControl": "Individual student control",
```

Comment:
This makes it look like I changed more than I did - I just alphabetized the strings that were out of order.